src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -44,6 +45,9 @@
   private static final int DELETION_INTERVAL_MAX = 60;
   private static final String HTTPS_ENABLED = "https-enabled";
   private static final String WRITER_QUEUE_SIZE = "writer-queue-size";
+  private static final String BATCH_METRICS_RETENTION_PERIOD = "batch-metrics-retention-period";
+  private static final long BATCH_METRICS_RETENTION_PERIOD_DEFAULT = 7;  // 7 minutes of metrics
+  private static final long BATCH_METRICS_RETENTION_PERIOD_LIMIT = 60;
```

Comment:
Can we add the time unit in the variable name such as
`BATCH_METRICS_RETENTION_PERIOD_DEFAULT_MINS` and so on ?

Reply:
Yes, that would be a good idea.